comixed-release/src/main/assembly/scripts/run.sh

```diff
@@ -112,7 +112,8 @@ fi
 # build a list of JVM arguments
 
 if [[ $LIBDIR ]]; then
-  JVMOPTIONS="${JVMOPTIONS} -classpath ${LIBDIR}"
+  JVMOPTIONS="$JVMOPTIONS -classpath"
+  CLASSPATH="${LIBDIR}"
 fi
 
-$JAVA ${JVMOPTIONS} -jar ${COMIXED_JAR_FILE} ${JAROPTIONS}
```

Comment:
This line fails when `LIBDIR` has a space in it (e.g. `/home/John Doe/comixed`) because the shell interprets the it as `-classpath`, `/home/John`, `Doe/comixed` (3 parameters).

I can't enclose `JVMOPTIONS` in quotes because then the shell interprets it as `-classpath /home/John Doe/comixed` (1 parameter, when in fact it should be 2).

See lines 116 and 119 in the patch for how I worked around this...